app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -2,11 +2,9 @@
 
 module Authentication
   module AuthnJwt
-
     PROVIDER_URI_RESOURCE_NAME = "provider-uri"
     JWKS_URI_RESOURCE_NAME = "jwks-uri"
     ISSUER_RESOURCE_NAME = "issuer"
-
+    JWT_ID_FIELD_NAME_VARIABLE = "token-app-property"
```

Comment:
the const name is very confusing...
Is it ID? Is it a field? field name? or variable?

Reply:
Its the variable name of the variable in charge of containg the field name of the identity in the JWT decoded token

Reply:
Renamed to IDENTITY_FIELD_VARIABLE